apps/deploy-web/src/services/feature-flag/index.ts

```diff
@@ -1,6 +1,7 @@
+import { getSession } from "@auth0/nextjs-auth0";
 import * as unleashModule from "@unleash/nextjs";
 
 import { serverEnvConfig } from "@src/config/server-env.config";
 import { FeatureFlagService } from "@src/services/feature-flag/feature-flag.service";
 
-export const featureFlagService = new FeatureFlagService(unleashModule, serverEnvConfig);
+export const featureFlagService = new FeatureFlagService(unleashModule, getSession, serverEnvConfig);
```

Comment:
thought(non-blocking): would be nice to mark ServerSide only services with some comment to make it explicit and easy to understand thanks to IDE hints:

```suggestion
/**
 * Server side only service
 */
export const featureFlagService = new FeatureFlagService(unleashModule, getSession, serverEnvConfig);
```